tcp/pcb.ml

```diff
@@ -465,6 +468,10 @@ struct
       Lwt.return_unit
     | None ->
       Tx.send_rst t id ~sequence ~ack_number ~syn ~fin
+    | exception ex ->
+      PCB_Log.err (fun f -> f "uncaught exception looking up handler for TCP port %d:@ %s"
+                      id.WIRE.local_port (Printexc.to_string ex));
+      Lwt.return_unit
```

Comment:
@talex5 as I mentioned in the mirage-net-xen issue, I'm in favour of failing hard (my goal is to not have any exceptions in the interface).  but let us do that a bit more organised: have the same semantics in mirage-net-unix and mirage-net-xen, and fix our own (well, MirageOS) libraries in the same release cycle.